src/Stratis.Bitcoin.Features.Consensus/CoinViews/DBreezeCoinView.cs

```diff
@@ -200,7 +200,9 @@ public Task SaveChangesAsync(IEnumerable<UnspentOutputs> unspentOutputs, IEnumer
                 {
                     transaction.ValuesLazyLoadingIsOn = false;
                     transaction.SynchronizeTables("BlockHash", "Coins", "Rewind");
-                    //transaction.Technical_SetTable_OverwriteIsNotAllowed("Coins"); // Why it was there and what is it for? No one knows.
+
+                    // Refers to issue #2483. https://github.com/stratisproject/StratisBitcoinFullNode/issues/2483
```

Comment:
probably better just mention what it does

Reply:
Issue is ok but also yeah feel free to just write here stuff even if its bulky